app/dashboard/models.py

```diff
@@ -1048,6 +1061,51 @@ def stats(self):
             (success_rate, 'Success Rate'),
         ]
 
+    @property
+    def get_quarterly_stats(self):
+        user_active_in_last_quarter = False
+        last_quarter = datetime.now() - timedelta(days=90)
+        bounties = self.bounties.filter(modified_on__gte=last_quarter)
+        loyalty_rate = 0
```

Comment:
looks like this is never computed?

Reply:
This is a part of `stats` property not `quarterly_stats` property.

I just fixed some linting errors in the `stats` property.

Reply:
Oops! Yes, you were right on this. Removed that bit out.